rules/do-you-know-the-3-steps-to-a-pbi/rule.md

```diff
@@ -1,74 +1,67 @@
 ---
 type: rule
-archivedreason: 
 title: Do you know the 3 steps to completing a PBI?
-guid: 1de9df77-9b69-4242-b648-e08e5980e9a6
 uri: do-you-know-the-3-steps-to-a-pbi
-created: 2013-08-30T06:33:21.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Eric Phan
-  url: https://ssw.com.au/people/eric-phan
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Matt Wicks
+    url: https://ssw.com.au/people/matt-wicks
+  - title: Piers Sinclair
+    url: " https://ssw.com.au/people/piers-sinclair"
 related: []
 redirects:
-- do-you-know-the-3-steps-to-completing-a-pbi
-
+  - do-you-know-the-3-steps-to-completing-a-pbi
+created: 2013-08-30T06:33:21.000Z
+archivedreason: null
+guid: 1de9df77-9b69-4242-b648-e08e5980e9a6
 ---
-
 The lifecycle of a PBI can be broken down into 3 steps:
 
 <!--endintro-->
 
+### 0. Before it's ready
+
+1. Received an email with some work to do? Get it into the backlog as a PBI ASAP (as per [Do you turn an email into a work item before starting work?](/turn-emails-into-work-items)).
+2. Is the PBI ready? Check your PBI against your  [Definition of Ready](/have-a-definition-of-ready). "Ready" PBIs must have Acceptance Criteria. A good Definition of Ready also encourages a test-first mentality in requirements e.g. Use Spec Flow (Given, When, Then) and/or create Test Cases and Test Steps first.
```

Comment:
"Ready" PBIs must have Acceptance Criteria. A good Definition of Ready also encourages a test-first mentality in requirements e.g. Use Spec Flow (Given, When, Then) and/or create Test Cases and Test Steps first."

Should this just be covered by the rule we are linking to? Otherwise it's a violation of DRY right?